src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb

```diff
@@ -43,6 +43,11 @@ Friend Module CompilationUtils
         If options Is Nothing Then
             options = TestOptions.ReleaseDll
         End If
+        
+        ' update the parseOption (which ensures the language versions a consistant     
```

Comment:
```suggestion
```

Don't think the comment is necessary here.

Reply:
The trailing whitespace is also what is failing the formatter.

Reply:
@jaredpar I think the comment is important, as the potential inconsistency isn't documented.
@333fred  It the GitHub online editor, that cause the issues around whitespace.

Reply:
This is a pretty common pattern in our code and virtually none of the other occurrences warrant a comment.